### PR TITLE
Aggregate swag shop blog posts into a developer tutorial.

### PR DESCRIPTION
**Pull request by prinzdezibel**
This PR resolves issue #543 
https://docs.reactioncommerce.com/reaction-docs/docs-543-prinzdezibel-create-swag-shop-tutorial/swag-shop-tutorial

### CODE REVIEW

File: redoc.json [hunk 1]
```diff
@@ -194,6 +194,36 @@
...
+      "alias": "swag-shop-collecting-requirements",
+      "label": "Collecting the shop requirements",
+      "repo": "reaction-docs",
+      "docPath": "developer/swag-shop/swag-shop-collecting-requirements.md"
+    }, {
+      "class": "guide-sub-nav-item",
+      "alias": "swag-shop-initialization",
+      "label": "Project scaffolding",
+      "repo": "reaction-docs",
+      "docPath": "developer/swag-shop/swag-shop-initilization.md"
```
Comment (machikoyasuda) [inline redoc.json]: `docPath` should be `developer/swag-shop/swag-shop-initialization.md`

File: developer/swag-shop/introduction.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Building & Launching a Swag Shop on Reaction
+
+---
+
+ **⚠ NOTICE:** This tutorial aggregates a couple of Reaction blog posts into one place that should serve our community as an accessible show case on how to build a fully customized shop on top of Reaction. The orginal blog posts can be found here:
+- <https://blog.reactioncommerce.com/building-and-launching-a-store-on-reaction/>
```
Comment (machikoyasuda) [inline developer/swag-shop/introduction.md]: Change this part to:
- [Building & Launching a Swag Shop on Reaction](https://blog.reactioncommerce.com/building-and-launching-a-store-on-reaction)
- [Landing Page](https://blog.reactioncommerce.com/building-launching-a-swag-shop-part-2/)
- [Landing Page, Part 2](https://blog.reactioncommerce.com/building-a-swag-shop-lp2/)
- [Category Tiles & Related Products](https://blog.reactioncommerce.com/building-a-swag-shop-category-tiles/)

File: developer/swag-shop/introduction.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Lately, the community team has been thinking a lot about how to provide guidance to our users when it comes to the task of choosing the right ecommerce software for their next project. One of our most requested topics on our Gitter chat is a request for real-life examples on how to customize a Reaction store according to a client's specification. It didn’t take long for us to follow up on this idea into something that serves the community.
+
+My intention with this tutorial is to take you with me on our journey. From concept and design to implementation and deployment, we’ll show you how to build a shop from start to finish, from backend to frontend, finally ending up with an operational shop!
+
+## Table of Contents
+
+- [Collecting the shop requirements](/developer/swag-shop/swag-shop-collecting-requirements.md)
```
Comment (machikoyasuda) [inline developer/swag-shop/introduction.md]: Add Part 1, Part 2, Part 3, Part 4 here like:

- [Part 1: Collecting the shop requirements](...)

File: developer/swag-shop/swag-shop-collecting-requirements.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+So we had another meeting with the product owner and explained the situation. When they realized that we were going to sell at most a few dozens article through our swag shop, but the Filter Bar would be the most expensive feature, they decided to postpone it to a possible later release of the shop. This is because a sophisticated filter mechanism is very useful for a shop offering hundreds or thousands of articles with different traits, but not so much for a shop with few dozens.
+
+In the end, we agreed to strip down the functional requirements for our swag shop a bit, which resulted in different tasks to be labeled as phase2 in GitHub's [issue tracker](https://github.com/reactioncommerce/reaction-swag-shop/issues).
+
+## What's next
+Read about how we implemented the [Landing Page](https://blog.reactioncommerce.com/building-launching-a-swag-shop-part-2/) in episode 2.
```
Comment (machikoyasuda) [inline developer/swag-shop/swag-shop-collecting-requirements.md]: Change this to the Docs tutorial link

File: redoc.json (hunk 1 above)
Comment (machikoyasuda) [reply redoc.json]: Let's also number these posts like:

Part 1: Collecting the shop requirements
Part 2: Project scaffolding
...

Comment (prinzdezibel) [reply redoc.json]: done

File: developer/swag-shop/introduction.md (hunk 2 above)
Comment (prinzdezibel) [reply developer/swag-shop/introduction.md]: The notice and the links are referring to the blog posts. Should we remove both together (the NOTICE and the link list?)

File: developer/swag-shop/introduction.md (hunk 3 above)
Comment (prinzdezibel) [reply developer/swag-shop/introduction.md]: done

File: developer/swag-shop/swag-shop-collecting-requirements.md (hunk 4 above)
Comment (prinzdezibel) [reply developer/swag-shop/swag-shop-collecting-requirements.md]: done.

**Pull request merged**